spec.bs

```diff
@@ -2235,6 +2247,37 @@ To <dfn>encode trusted signals keys</dfn> given an [=ordered set=] of [=strings=
 
 </div>
 
+A <dfn>bidding signals per interest group data</dfn> is a [=struct=] with the following
+[=struct/items=]:
+
+<dl dfn-for="bidding signals per interest group data">
+  : <dfn>updateIfOlderThanMs</dfn>
+  :: Null or a {{double}}. If set, it is the [=duration=] in milliseconds that specifies that the
+    [=interest group=] should be [updated](#interest-group-updates) if it hasn't been joined or
+    updated within that duration.
+</dl>
+
+<div algorithm>
+
+To <dfn>append to a bidding signals per-interest group data map</dfn> given an [=ordered map=]
+|sourceMap|, an [=ordered set=] |igNames|, and an [=ordered map=] |destinationMap|:
+1. [=map/For each=] |sourceKey| → |sourceValue| of |sourceMap|:
+  1. If |igNames| does not [=set/contain=] |sourceKey|, [=iteration/continue=].
+  1. Let |perGroupData| be a new [=bidding signals per interest group data=].
+  1. TODO: handle priority vector.

```

Comment:
I realize this isn't added by this PR since it looks like it's moved, but please file a bug and link to it here just like we would in the Chromium code base.

Reply:
We stopped using the format of `Issue(WICG/turtledove#1144)` which you are using here, which uses GitHub API to fetch the issue (there's one usage left, which we can get rid of as well).
Instead, you can do something similar to [this](https://github.com/WICG/turtledove/blob/main/spec.bs#L5350).
`Issue: TODO: bla bla bla about the todo
    (<a href="https://github.com/WICG/turtledove/issues/1144">WICG/turtledove#1144</a>)`, where the "WICG/..." here is just some plain text, for the given link. It does not use GitHub API.

Reply:
Done, and I mailed https://github.com/WICG/turtledove/pull/1146 for migrating that other issue.